hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/NSSummaryTask.java

```diff
@@ -121,6 +130,8 @@ public Pair<String, Boolean> reprocess(OMMetadataManager omMetadataManager) {
         .reprocessWithFSO(omMetadataManager));
     tasks.add(() -> nsSummaryTaskWithLegacy
         .reprocessWithLegacy(reconOMMetadataManager));
+    tasks.add(() -> nsSummaryTaskWithOBS
+        .reprocessWithOBS(reconOMMetadataManager));
 
     List<Future<Boolean>> results;
     ExecutorService executorService = Executors
```

Comment:
We can think of increasing the fixed number of threads to 3, if want to run in parallel, but this may back fire as well because resources in recon are limited, however it will be good if we can analyse and test the resource usage with minimum of 10 million add key events or test with limited number of events depending on availability of system resources to have an idea how Recon will perform. Else after finishing of previous 2 tasks, this OBS taks will run.